CHANGELOG.md

```diff
@@ -10,7 +10,131 @@ tries to adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Added
 
-- Started keeping a changelog!
+
+## [0.44.12] - 2025-04-25
+
+### Added
+
+- [#2039](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2039) Add generic `shift_left!` and `shift_right!` for univariate polynomials
+- [#2047](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2047) Add `falling_factorial`
+
+### Changed
+
+- [#2055](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2055) Add comments and more solving examples
+- [#2058](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2058) Improve univariate conversion
+- [#2057](https://github.com/Nemocas/AbstractAlgebra.jl/pull/2057) Fix a JET issue in `iterate(::ProductIterator)`
```

Comment:
```suggestion
```
this won't ever affect any user, it's just fixes something that JET/the compiler couldn't infer